services/worker/src/worker/job_runners/dataset_info.py

```diff
@@ -51,24 +61,31 @@ def __init__(self, message: str, cause: Optional[BaseException] = None):
         super().__init__(message, HTTPStatus.INTERNAL_SERVER_ERROR, "PreviousStepFormatError", cause, False)
 
 
-def compute_dataset_info_response(dataset: str) -> DatasetInfoResponse:
+def compute_dataset_info_response(dataset: str) -> Tuple[DatasetInfoResponse, float]:
     """
-    Get the response of /dataset-info for one specific dataset on huggingface.co.
+    Get the response of dataset-info for one specific dataset on huggingface.co.
     Args:
         dataset (`str`):
             A namespace (user or an organization) and a repo name separated
             by a `/`.
     Returns:
-        `DatasetInfoResponse`: An object with the dataset_info response.
+        (`DatasetInfoResponse`, `float`): Tuple of an object with the dataset_info response and
+        progress float value from 0. to 1. which corresponds to the percentage of dataset configs
+        correctly processed and included in current response (some configs might not exist in cache yet
+        or raise errors).
     <Tip>
     Raises the following errors:
         - [`~job_runners.dataset_info.PreviousStepStatusError`]
-          If the previous step gave an error.
+            If the previous step gave an error.
         - [`~job_runners.dataset_info.PreviousStepFormatError`]
-            If the content of the previous step has not the expected format
+            If the content of the previous step doesn't have the expected format.
+        - [`~libcommon.dataset.DatasetNotFoundError`]
+            If the dataset does not exist, or if the
+            token does not give the sufficient access to the dataset, or if the dataset is private
+            (private datasets are not supported by the datasets server)
     </Tip>
     """
-    logging.info(f"get dataset_info for dataset={dataset}")
+    logging.info(f"get dataset_info for {dataset=}")
```

Comment:
```suggestion
    logging.info(f"get dataset_info for {dataset}")
```

Reply:
don't you like this syntax? i think it's convenient to use `f"{dataset=}"` instead of `f"dataset={dataset}"`) not to duplicate words.

Reply:
Oh, I had no idea that it was a valid syntax, I thought it was a typo. Sorry!!! And thanks for teaching me that.

Reply:
Doc is here: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging